### ISSUE TITLE
Dokku reinstalls Ruby, nginx and Jekyll every time I deploy

### ISSUE THREAD

**Issue by xSke** [inket/dokku-buildpack-jekyll3-nginx#3]
It reinstalls everything, every time I deploy. This is painfully slow.

Log:
    $ git push origin master
    Counting objects: 7, done.
    Delta compression using up to 4 threads.
    Compressing objects: 100% (6/6), done.
    Writing objects: 100% (7/7), 985 bytes | 0 bytes/s, done.
    Total 7 (delta 3), reused 0 (delta 0)
    -----> Cleaning up...
    -----> Building [redacted] from herokuish...
    -----> Adding BUILD_ENV to build environment...
    -----> Fetching custom buildpack
    -----> Jekyll app detected
    -----> Installing Ruby 2.2.1
    remote: Cloning into '/app/.rbenv'...
    remote: Cloning into '/app/.rbenv/plugins/rvm-download'...
           Download and extract ruby 2.2.1 from the RVM repository
           Ruby 2.2.1 has been installed
    -----> Fetching nginx binaries
    -----> Vendoring nginx 1.6.0
    -----> Using default nginx.conf.erb
    -----> Using default mime.types
           Jekyll not found!
    -----> Installing Jekyll
           Successfully installed liquid-2.2.2
           1 gem installed
           Successfully installed colorator-0.1
           Successfully installed sass-3.4.20
           Successfully installed jekyll-sass-converter-1.4.0
           Successfully installed rb-fsevent-0.9.7
           Building native extensions.  This could take a while...
           Successfully installed ffi-1.9.10
           Successfully installed rb-inotify-0.9.5
           Successfully installed listen-3.0.5
           Successfully installed jekyll-watch-1.3.0
           Successfully installed kramdown-1.9.0
           Successfully installed liquid-3.0.6
           Successfully installed mercenary-0.3.5
           Successfully installed rouge-1.10.1
           Successfully installed safe_yaml-1.0.4
           Successfully installed jekyll-3.0.1
           Building native extensions.  This could take a while...
           Successfully installed redcarpet-3.3.4
           Successfully installed sass-3.4.20
           16 gems installed
           Successfully installed libv8-3.16.14.13-x86_64-linux
           Successfully installed ref-2.0.0
           Building native extensions.  This could take a while...
           Successfully installed therubyracer-0.12.2
           3 gems installed
    -----> Building Jekyll site
           Configuration file: /tmp/build/_config.yml
           Source: /tmp/build
           Destination: /tmp/build/_site
           Incremental build: disabled. Enable with --incremental
           Generating...
           done in 0.439 seconds.
           Auto-regeneration: disabled. Use --watch to enable.
    -----> Discovering process types
           Default types for  -> web
    -----> Releasing [redacted](dokku/[redacted]:latest)...
    -----> Deploying [redacted](dokku/[redacted]:latest)...
    -----> DOKKU_SCALE file found
    =====> web=1
    -----> Running pre-flight checks
           For more efficient zero downtime deployments, create a file CHECKS.
           See http://progrium.viewdocs.io/dokku/checks-examples.md for examples
           CHECKS file not found in container: Running simple container check...
    -----> Waiting for 10 seconds ...
    -----> Default container check successful!
    =====> [redacted] container output:
    =====> end [redacted] container output
    -----> Running post-deploy
    -----> Configuring [redacted]...
    -----> Creating http nginx.conf
    -----> Running nginx-pre-reload
           Reloading nginx
    -----> Shutting down old containers in 60 seconds
    =====>  [redacted]
    =====> Application deployed:
           [redacted]

```
To dokku@[redacted]
   [redacted]  master -> master
```

**Comment by inket** [comment]
Indeed, but AFAIK there's not much we can do because Dokku creates a new container for each deploy.

The ruby install itself only takes around 5 seconds, and the website never goes down thanks to zero downtime deployments. I don't see this as a big issue.
